job-service-db/src/main/resources/v2.7.0/changelog.xml

```diff
@@ -105,4 +105,47 @@
                          schemaName="public">
         </createProcedure>
     </changeSet>
+    <changeSet id="update_procedure_get_job_last_update_time" runOnChange="true" author="Gamma Team">
+        <createProcedure path="procedures/getJob.sql"
+                         procedureName="get_job"
+                         schemaName="public">
+        </createProcedure>
+    </changeSet>
+    <changeSet id="update_procedure_get_jobs_last_update_time" runOnChange="true" author="Gamma Team">
+        <createProcedure path="procedures/getJobs.sql"
+                         procedureName="get_jobs"
+                         schemaName="public">
+        </createProcedure>
+    </changeSet>
+    <changeSet id="update_procedure_create_job_last_update_time" runOnChange="true" author="Gamma Team">
+        <createProcedure path="procedures/createJob.sql"
+                         procedureName="create_job"
+                         schemaName="public">
+        </createProcedure>
+    </changeSet>
+    <changeSet id="update_procedure_cancel_job_last_update_time" runOnChange="true" author="Gamma Team">
+        <createProcedure path="procedures/cancelJob.sql"
+                         procedureName="cancel_job"
+                         schemaName="public">
+        </createProcedure>
+    </changeSet>
+    <changeSet id="update_procedure_create_job_for_dependencies_last_update_time" runOnChange="true" author="Gamma Team">
+        <createProcedure path="procedures/createJobForDependencies.sql"
+                         procedureName="create_job"
+                         schemaName="public">
+        </createProcedure>
+    </changeSet>
+    <changeSet id="update_procedure_internal_report_task_status_last_update_time" runOnChange="true" author="Gamma Team">
+        <createProcedure path="procedures/reportTaskStatus.sql"
+                         procedureName="internal_report_task_status"
+                         schemaName="public">
+        </createProcedure>
+    </changeSet>
+    <changeSet id="update_job_table_add_column_last_update_time" author="Gamma Team">
```

Comment:
Does this not need to come before the procedure updates or does it not matter?

Reply:
Either way round, there'll be a period of time where the schema and procedure are mismatched, and procedure calls will fail.  As I now understand it, upgrades are expected to have downtime - stop database-using services, upgrade the database, start database-using services - so it seems to me that it shouldn't matter.